src/plugins/album-actions/index.ts

```diff
@@ -159,26 +159,26 @@ export default createPlugin<
       let playlistButtons: NodeListOf<HTMLElement> | undefined;
       const playlist =
         document.querySelector('ytmusic-playlist-shelf-renderer') ??
-        document.querySelector('ytmusic-shelf-renderer');
+        [...document.querySelectorAll('ytmusic-shelf-renderer')].at(-1);
```

Comment:
Have you tried CSS pseudo selectors, e.g. `:last`, and does it not work?
If it does not work, then I suggest the below patch to avoid allocating an array.

```suggestion
        Array.prototype.at.call(document.querySelectorAll('ytmusic-shelf-renderer'), -1)!;
```

Reply:
Thanks!
Regarding as to why I did not use pseudo selectors, these playlists are in different parts of the DOM.